Contributing.md

```diff
@@ -53,21 +56,35 @@ Welcome to the CARMAPlatform contributing guide. There are a few guidelines that
 * When you have finished making your changes, you'll need to push up your changes to a topic branch in your fork of the repository.  Read the article ["Committing and reviewing changes to your project"](https://help.github.com/desktop/guides/contributing-to-projects/committing-and-reviewing-changes-to-your-project/) on GitHub. 
 * With them all committed, push them, this pushes everything in that branch up. 
 
-### Submitting Changes
+#### Submit Changes
 **Let you tell others about changes you've pushed to a GitHub branch.**
 * A Pull Request is the way to notify the project maintainers that you have some work that they should review and add to the project. 
 * You’re requesting that they pull your changes in. Read the article ["Using Pull Requests"](https://help.github.com/articles/using-pull-requests) on GitHub.
 * Then you can go back to your forked branch and issue a pull request to the repository and choose the correct original branch (should be **`develop`**).
 * To create one, go to your fork of the project, click on the Pull Requests tab, and click the big green “New Pull Request” button.
 
-### Merging Changes
+#### Merge Changes
 **We want your help to make CARMAPlatform Project great.**
 * All pull requests will be reviewed by the CARMAPlatform team. 
 * During the review of your pull request the team member will either merge it, request changes to it, or close it with an explanation. 
 * For major changes the reviewer may require additional support from the team, which could cause some delay. 
 * We'll do our best to provide updates and feedback throughout the process. 
 * Feel free to open pull requests, and the CARMAPlatform team will communicate through it with any comments.
 
+#### Become a Collaborator
+**Please fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSdjEh0OPBVw-jaqfZ6gbyoPV0RetMOirvzp5NpE_whdrnhLdg/viewform) to be added to the CARMA Collaborative.**
+
+Four Contribution Levels
+1.	Informative Group Contributor.
+This level is for stakeholders interested at an informational level to keep up-to-date on CARMA activities advancing CDA. This level receives CARMA news, product updates, and first access to public CARMA events.
+2.	Prospective User.
+This level is for stakeholders who are interested in contributing to CARMA research, development, and testing in the future. This level receives the benefits of Informative Group Contributors, and additionally, this level may request support to prepare to contribute to CARMA and will receive first access to select CARMA group events.
+3.	User.
+This level is for contributors who have contributed code and/or have a vehicle capable of utilizing CARMA. These contributors will also likely be using ROS Discourse for advancing CDA through open and active collaboration. This level receives the benefits of Prospective Users as well as advanced support and access to technical working groups.

```

Comment:
I suggest adding a hyperlink for our ROS discourse page where it is mentioned in this line.